ctod.dd

```diff
@@ -275,8 +276,8 @@ $(H3 <a name="arrayloop">Looping through an array</a>)
 
 $(H4 The C Way)
 $(P
-       The array length is defined separately, or a clumsy
-       sizeof() expression is used to get the length.)
+       The array length is defined separately, or a clumsy and error prone
```

Comment:
the very common idiom `sizeof array/sizeof array[0]` breaks when the `array` expression decays to a pointer as arrays often do in C.